acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueGetSignersRpcTest.java

```diff
@@ -37,21 +37,21 @@ public void setUp() throws Exception {
   @Test
   public void shouldBeAbleToGetValidatorsForBlockNumber() {
     cluster.verify(clique.validatorsAtBlockEqual("0x0", minerNode1));
-    minerNode1.waitUntil(wait.chainHeadIsAtLeast(1));
+    minerNode1.verify(blockchain.minimumHeight(1));
 
     minerNode1.execute(cliqueTransactions.createAddProposal(minerNode2));
-    cluster.waitUntil(wait.chainHeadHasProgressedByAtLeast(minerNode1, 1));
+    cluster.verify(blockchain.minimumHeightProgression(minerNode1, 1));
```

Comment:
progression sounds a bit odd here. Is there a consistent phrasing or tense used or meant to be used for the dsl?

Reply:
suggested alternative?

Reply:
`blockchain.reachesHeight()`